actors/miner/src/types.rs

```diff
@@ -331,6 +334,39 @@ pub struct SectorOnChainInfo {
     pub sector_key_cid: Option<Cid>,
 }
 
+impl SectorOnChainInfo {
+    pub fn expires_early(&self) -> bool {
+        self.proof_expiration < self.commitment_expiration
+    }
+
+    pub fn expires_at(&self) -> ChainEpoch {
+        min(self.proof_expiration, self.commitment_expiration)
+    }
+
+    pub fn possible_proof_expirations(&self, now: ChainEpoch) -> Vec<ChainEpoch> {
+        let delta_e = network::EPOCHS_IN_YEAR; // TODO correct value
```

Comment:
Take the max as an argument and fix up this method. Please add unit tests (i.e. in this file)

Reply:
I need to delete this method. I don’t use it after all.